src/java.desktop/share/classes/sun/awt/AppContext.java

```diff
@@ -254,26 +245,20 @@ public ClassLoader run() {
     private static final ThreadLocal<AppContext> threadAppContext =
             new ThreadLocal<AppContext>();
 
-    @SuppressWarnings("removal")
+    //@SuppressWarnings("removal")
```

Comment:
`@SuppressWarnings("removal")` is commented out? Probably meant to remove it?

Reply:
ok